MvvmCross-iOSSupport/MvvmCross.iOS.Support.XamarinSidebar/MvxSidebarPresenter.cs

```diff
@@ -33,94 +32,77 @@ protected virtual bool PresentationHintHandler(MvxPanelPresentationHint hint)
             return true;
         }
 
-        public override void Show(MvxViewModelRequest request)
+        protected override void RegisterAttributeTypes()
         {
-            IMvxIosView viewController = Mvx.Resolve<IMvxIosViewCreator>().CreateView(request);
-            Show(viewController);
+            base.RegisterAttributeTypes();
+
+            _attributeTypesToShowMethodDictionary.Add(
+                typeof(MvxSidebarPresentationAttribute),
+                (vc, attribute, request) => ShowSidebarViewController(vc, attribute as MvxSidebarPresentationAttribute, request));
```

Comment:
That's my point. I couldn't oversee if the attribute could be null or not. If it can't be null then we should do a direct cast. 

Besides the performance difference (which is minimal), the as operator still suggests it could be null, which could lead other developers on the wrong path (of which this is an example 😝)